engineering_tools/carma_script_extensions/cloud-sim

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-#  Copyright (C) 2022 LEIDOS.
+#  Copyright (C) 2022-2023 LEIDOS.
```

Comment:
The help output for this script should reference the other configure-cloud-sim.bash to tell users that one needs to be run first,  I think.